src/main/webapp/edit.jsp

```diff
@@ -19,22 +19,26 @@
     </style>
 
     <%
-        CustomData cd = CustomData.getModulePersonalizationData(pageContext);
-        String numberOfActivities = cd.getValue("numberOfActivities");
-
         Configuration configuration = ConfigUtil.loadConfig();
+        CustomData cd = CustomData.getModulePersonalizationData(pageContext);
 
         int maxNumberOfActivities = configuration.getMaxNumberOfEvents();
 
-        if (numberOfActivities == null) {
-            numberOfActivities = String.valueOf(configuration.getDefaultNumberOfEvents());
+        int numberOfActivities;
+
+        if (cd.getValue("numberOfActivities") == null) {
+            numberOfActivities = configuration.getDefaultNumberOfEvents();
+        } else {
+            numberOfActivities = Math.min(Integer.parseInt(cd.getValue("numberOfActivities")),
```

Comment:
Mjah, als dat profiel corrupt is kom je d'r nooit meer uit. Ook doe je 2x die `cd.getValue()`. Kunnen we niet gewoon zoiets doen:

```
Integer numberOfActivities = null;

try { 
  numberOfActivities = Integer.parseInt(cd.getValue("numberOfActivities")
}
catch(NumberFormatException ignored) {}

if(numberOfActivities == null || configuration.getMaxNumberOfEvents() < numberOfActivities) {
    numberOfActivities  = configuration.getDefaultNumberOfEvents();
}
```